.github/workflows/beam_PerformanceTests_ManyFiles_TextIOIT.yml

```diff
@@ -0,0 +1,91 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: PerformanceTests ManyFiles TextIOIT
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    - cron: '10 2 * * *'
```

Comment:
should be `10 2/14 * * *` in accordance with the Jenkins pattern

Reply:
fixed `10 2/12 * * *`  every 12 hour (jenkins pattern)